frontend/packages/console-shared/src/utils/pod-ring-utils.ts

```diff
@@ -22,6 +22,25 @@ const applyPods = (podsData: PodRingData, dc: PodRCData) => {
   return podsData;
 };
 
+export const podRingLabel = (resourceObj, isScalingAllowed) => {
+  const title =
+    resourceObj.status.availableReplicas || (isScalingAllowed ? 'Scaled to 0' : 'AutoScaled to 0');
+
+  const subTitle =
+    resourceObj.spec.replicas !== resourceObj.status.availableReplicas
+      ? !resourceObj.spec.replicas
+        ? `pods`
+        : `scaling to ${resourceObj.spec.replicas}`
+      : resourceObj.spec.replicas > 1 || resourceObj.spec.replicas === 0
+      ? 'pods'
+      : 'pod';
```

Comment:
can we have it like below

```
!resourceObj.spec.replicas ? 
   `pods`: resourceObj.spec.replicas !== resourceObj.status.availableReplicas ? 
         `scaling to ${resourceObj.spec.replicas}` :   `pods`;
```

and for `resourceObj.spec.replicas > 1 || resourceObj.spec.replicas === 0 ? 'pods': 'pod'` when  will this be false

Reply:
ah when `resourceObj.spec.replicas ===1`